testing/php7-memcached/APKBUILD

```diff
@@ -5,7 +5,7 @@ _pkgreal=memcached
 _pkgrepo=php-memcached
 _pkgbranch=php7
 _pkgcommit=583ecd68faec886ac9233277531f78fb6e2043c7
-pkgver=3.0.0pre20160808
+pkgver=3.0.0_pre160808
```

Comment:
I’ve omitted the last digit for the purpose. What if the upstream someday release 3.0.0_beta? Then you will have a problem, because `3.0.0_beta < 3.0.0_pre160808`. But `3.0.0_beta > 3.0_pre160808`.